pkg/routes/context.go

```diff
@@ -31,6 +32,18 @@ func (c *Ctx) WithUrls() *Ctx {
 	return c
 }
 
+func (c *Ctx) WithSession() *Ctx {
+	data := fiber.Map{}
+
+	user, _ := session.GetActiveUsername(c.Ctx)
```

Comment:
Understand why this handling is not needed and maybe add a comment explaining why its not handled
```
	user, _ := session.GetActiveUsername(c.Ctx)
```